<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1001</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainLayout" rowstretch="1,1,1,1,1,1,1,1" columnstretch="1,1,1,1,1,1,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="8">
      <widget class="QGroupBox" name="playlistGBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>122</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="playlistVLayout" stretch="1,1,1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="connectBt">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createPlaylistBt">
           <property name="text">
            <string>Create playlist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="playlistNameTxEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="playlistViewGBox">
           <property name="title">
            <string>User Playlists</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>29</y>
              <width>121</width>
              <height>431</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="playGboxVLayout" stretch="0">
             <item>
              <widget class="QPushButton" name="playBt">
               <property name="text">
                <string>Play</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1" colspan="7">
      <widget class="QGroupBox" name="searchGBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Search on spotify</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>851</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="searchHLayout" stretch="1,1,1,1">
         <item>
          <widget class="QTextEdit" name="searchTxEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="artistRBt">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Artist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="musicRBt">
           <property name="text">
            <string>Music</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBt">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="removeTrackBt">
       <property name="text">
        <string>Remove track</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="6" colspan="3">
      <widget class="QGroupBox" name="tracksGBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Playlist tracks</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>361</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="tracksVLayout"/>
       </widget>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="addTrackBt">
       <property name="text">
        <string>&lt;-- Add track</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="7">
      <widget class="QGroupBox" name="logGBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output log</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>851</width>
          <height>72</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="logHLayout" stretch="0">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="logPTxEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="5" rowspan="6" colspan="3">
      <widget class="QGroupBox" name="resultGBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Search Results</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>351</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="resultVLayout" stretch=""/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
